src/sass/user-profile.scss

```diff
@@ -57,7 +57,13 @@
   }
 
   .card {
-    .resume-saved-application-container {
+    p {
```

Comment:
Should make a class for this.

Reply:
Have relocated this in `.saved-form-metadata-item:last-child`